lessons/08-tests-tasks/README.md

```diff
@@ -174,11 +195,6 @@ We've built a testing framework for the cat app with a few example tests (the ap
   - Challenge: make a copy of the `inclass` directory and replace npm with a different task runner (e.g. grunt or gulp). What's different? Easier? Harder?
 - Many editors support code quality plugins! Try adding a linter (jshint is an option, as are eslint and jslint) to your editor of choice.
 
-## Feedback
-We'd love your feedback on today's class -- fill out [this survey](http://goo.gl/forms/uZPfjZCGFf) if you'd like.
-
-If you're done, start the homework!
-
 # Before Class 9
 ### Assignment
-Using the preclass and inclass exercises as examples, set up a test environment and write tests for Twoter or for the burger app. Then fill out the [Homework Survey]().
\ No newline at end of file
+Using the preclass and inclass exercises as examples, set up a test environment and write tests for Twoter. Then fill out the [Homework Survey]().
```

Comment:
Here's the homework survey: https://docs.google.com/forms/d/e/1FAIpQLSfZz_oUjcO5IWfwdq82ucR7_fZCiw_eVbvJlXcPdpiXY3rq0A/viewform